Dockerfile.dependencies

```diff
@@ -0,0 +1,25 @@
+# This image will be published as dspace/dspace-dependencies
+# The purpose of this image is to make the build for dspace/dspace run faster 
+
+# Step 1 - Run Maven Build
+FROM maven:3-jdk-8 as build
```

Comment:
Does that mean we always build on JDK8, even if we want to test with JDK7? (Yes, I know its eol)

Reply:
@pnbecker , the DSpace 5 Dockerfiles offer a jdk7 and jdk8 options since those jdks are referenced in the installation notes.  For DSpace 6, only the jdk8 Dockerfiles are offered.  New Dockerfiles would need to be added if that support is necessary.